speedup_import.md

```diff
@@ -0,0 +1,99 @@
+# Speedup Import
```

Comment:
In the long run I don't think we want this file at the top level.  But it should be fine for now (until we are happy we have the issue resolved).